### ISSUE TITLE
[Feature] Native desktop app wrapper to launch and manage smtp4dev

### ISSUE THREAD

**Issue by rnwood** [rnwood/smtp4dev#675]
I've noticed that quite a lot of people are still preferring the old v2 Windows Forms desktop app.

So the bridge the gap, I'd like to provide a desktop app (Windows, MacOS, Linux) that launches and manages smtp4dev and makes it sit in the system tray in the same way that v2 did. It will use [Chromely](https://github.com/chromelyapps/Chromely) which in turn uses the Chromium Embeeded Framework - CEF) for the browser bit so that it's not a resource hog, like it would be if it were an Electron app!

This will be an optional and easy way to launch and manage smtp4dev for those who are scared off by launching from the command line, Docker, or dotnet global tool. Since it will add ~250MB, there will be separate downloads available for this.

**Comment by steskalja** [comment]
Maybe look at Maui or Avalonia framework to make an actual Native app that is not a web wrapper.

**Comment by rnwood** [comment]
Yes, using those would be possible, but I'm not really interested in maintaining two full UIs.

The point is people continuing to use the old V2 app seem to want a single file they can just double click on and have the server start and UI come up, without needing to pick a port number for the web server(with potential for conflicts) and then open their browser. Using a hybrid launcher approach can solve that bit without the maintenance overhead adding a full (cross platform) desktop UI.
